browser/src/DownloadsPage/DownloadsPage.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import { describe, expect, test } from '@jest/globals'
+import 'jest-styled-components'
+
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+import DownloadsPage from './DownloadsPage'
+
+import { withDummyRouter } from '../../../tests/__helpers__/router'
+
+// TODO: (rgrant) add mocked data for each of the different types of layout effects on line 44 of DownloadsPage.tsx
+test('Downloads Page has no unexpected changes', () => {
```

Comment:
@rileyhgrant there's an easy way to do this, remember that these specs are just ordinary code, so something like this ought to work:

```
const locationPrefixes = ['#v2-liftover', '#v3-', '#exac-', '#research-']

locationPrefixes.forEach((locationPrefix) => {
  test(`Downloads page with ${locationPrefix} location prefix`, () => {
    const tree = renderer.create(withDummyRouter(<DownloadsPage location={{ hash: locationPrefix }} />))
    expect(tree).toMatchSnapshot()
  })
})
```

...and you get four tests for the price of one.

Reply:
Aha - ok that makes sense. Thanks!